provision/provision.sh

```diff
@@ -282,7 +282,10 @@ profile_setup() {
   fi
 
   echo " * Copying /srv/config/ssh_known_hosts to /etc/ssh/ssh_known_hosts"
- cp -f /srv/config/ssh_known_hosts /etc/ssh/ssh_known_hosts
+  cp -f /srv/config/ssh_known_hosts /etc/ssh/ssh_known_hosts
+  echo " * Enabling SSH Password Authentication for database applications"
+  echo "PasswordAuthentication yes" >> /etc/ssh/sshd_config
```

Comment:
oh dear, I realise now this file is going to get very large with the same line repeated over and over again

Reply:
maybe with `cat /etc/ssh/sshd_config | grep 'PasswordAuthentication yes'``we can do a check if it is already on it